lib/ast/lint.js

```diff
@@ -4,24 +4,84 @@ var ASTVisitor = require('juttle/lib/compiler/ast-visitor');
 var _ = require('underscore');
 
 var Lints = {
-    lint: function(ast) {
-        this.visit(ast);
+    lint: function(ast, o) {
+        if (!o.nameField) { throw new Error('Missing required option: nameField'); }
+
+        var options = {
+            nameFields: 0,
+            disallowNameField: false,
+            nameField: o.nameField
+        };
+
+        this.visit(ast, options);
+
+        if (options.nameFields > 1) {
```

Comment:
I think this would be more readable if you called it `nameFilters` instead of `nameFields`.